calamus/fields.py

```diff
@@ -86,40 +92,35 @@ def __str__(self):
 class _JsonLDField(fields.Field):
     """Internal class that enables marshmallow fields to be serialized with a JsonLD field name."""
 
-    def __init__(self, field_name, *args, **kwargs):
+    def __init__(self, field_name=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.field_name = field_name
 
         self.reverse = kwargs.get("reverse", False)
+        self.init_name = kwargs.get("init_name", None)
 
     @property
     def data_key(self):
         """Return the (expanded) JsonLD field name."""
+        if self.field_name is None:
+            raise ValueError("field_name was not set for {}".format(self))
         return str(self.field_name)
 
     @data_key.setter
     def data_key(self, value):
         pass
 
     def _deserialize(self, value, attr, data, **kwargs):
-        value = normalize_value(value)
+        if not isinstance(self, fields.List):
```

Comment:
we could call `super(fields.List, self)._deserialize()` in our `List` class directly, bypassing the `_deserialize` on `_JsonLDField`, then we don't need to do the instance check here.